addons/website/data/website_contactus_data.xml

```diff
@@ -0,0 +1,174 @@
+                                                        <div class="col-sm">
+                                                            <input id="contact7" type="hidden" class="form-control s_website_form_input" name="email_to"/>
+                                                        </div>
+                                                    </div>
+                                                </div>
+                                                <div class="form-group col-12 s_website_form_submit" data-name="Submit Button">
+                                                    <div style="width: 200px;" class="s_website_form_label"/>
+                                                    <a href="#" role="button" class="btn btn-primary btn-lg s_website_form_send">Submit</a>
+                                                    <span id="s_website_form_result"></span>
+                                                </div>
+                                            </div>
+                                        </form>
+                                    </div>
+                                </section>
+                            </div>
+                            <div class="col-lg-4 mt-4 mt-lg-0">
+                                <address itemscope="itemscope" itemtype="http://schema.org/Organization">
+                                    <p class="p-0 m-0">My Company</p>
+                                    <ul class="list-unstyled mb-0 pl-2">
+                                        <li><i class="fa fa-map-marker fa-fw mr-2" /><span class="o_force_ltr">Chaussée de Namur 40</span></li>
+                                        <li><i class="fa fa-phone fa-fw mr-2" /><span class="o_force_ltr">+ 32 81 81 37 00</span></li>
+                                        <li><i class="fa fa-1x fa-fw fa-envelope mr-2" /><span>info@yourcompany.example.com</span></li>
+                                    </ul>
+                                </address>
+                            </div>
+                        </div>
+                    </div>
+                </section>
+            </div>
+        </t>
+    </template>
+
+    <record id="website.contactus_thanks" model="website.page">
+        <field name="name">Thanks (Contact us)</field>
+        <field name="type">qweb</field>
+        <field name="url">/contactus-thank-you</field>
+        <field name="website_indexed" eval="False"/>
+        <field name="is_published">True</field>
+        <field name="key">website.contactus_thanks</field>
+        <field name="arch" type="xml">
+            <t name="Thanks (Contact us)" t-name="website.contactus_thanks">
+                <t t-call="website.layout">
+                    <div id="wrap" class="oe_structure">
+                        <section class="s_text_block pt40 pb40 o_colored_level " data-snippet="s_text_block">
+                            <div class="s_allow_columns container mt-4 mb-5">
```

Comment:
remove the `mt-4 mb-5`, this is creating a double spacing as there is already a spacing on the line above (section pt40 & pb40) + margin can't be changed by editor